cmd/derper/derper.go

```diff
@@ -132,9 +119,22 @@ func writeNewConfig() config {
 	return cfg
 }
 
+func cancelOnSignal(cancelf func()) {
+	sigs := make(chan os.Signal, 1)
+	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
+	go func() {
+		<-sigs
```

Comment:
Nit: this does block forever if we never get a signal (which is probably fine?); maybe worth doing something like `func withSignal(context.Context, sigs ...os.Signal)`, so that we can also select on `ctx.Done`?

Reply:
done